tailwind.config.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
                 60: "#825fff",
```

Comment:
`violet-90` is missing in the colour palette but is needed for the progress text color
```suggestion
                90: "#443185",
                70: "#6449c4",
                60: "#825fff",
```

Reply:
83da75b